app/javascript/packs/filtering.js

```diff
@@ -0,0 +1,17 @@
+$(function(){
+  $("#search_box").keyup(function(){
+    var myInput = $(this).val().toLowerCase();
```

Comment:
jQueryオブジェクトは `$` を付けるのが一般的です。

```suggestion
    var $myInput = $(this).val().toLowerCase();
```

また，その後の処理で何度も出てきているオブジェクトを変数にしておいた方がよいでしょう。

```
    var $lessonCard = $("#top-wrapper div.lesson-card")
```

余裕があれば，`var`や `function` のような古い書き方でなく `ES6` の書き方が使えるとなおよいですね。

Reply:
ES6での書き方にトライします。
ES5 でも良いではないかと思いがちですが、ES6 の書き方を知っておく事で
Qiita記事などで、ES5 で書かれているのか、ES6で書かれているのかの選球眼を身に付けることが
できる、ということを見過ごせないですよね。